psi4/driver/p4util/__init__.py

```diff
@@ -37,3 +37,4 @@
 from .p4regex import *
 from .python_helpers import *
 from .solvers import *
+from .writer import *
```

Comment:
try `from . import writer` since you don't actually need any functions, just need the code executed. I think that'll appease lgtm.